### PR TITLE
Added a translated code of conduct

### PR DESCRIPTION
**Pull request by alexzimmer96**
This PR adds a german version of the Code of Conduct as described in #18601

### CODE REVIEW

File: content/de/community/code-of-conduct.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+cid: community
+css: /css/community.css
+---
+
+<div class="community_main">
+<h1>Kubernetes Community Code of Conduct</h1>
+
+Kubernetes folgt dem
+<a href="https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/de.md">CNCF Verhaltenskodex</a>.
+Der Kodex findet sich weiter unten auf der Seite, wie er auch in
```
Comment (mkorbi) [inline content/de/community/code-of-conduct.md]: Der Kodex befindet sich ...

File: content/de/community/static/cncf-code-of-conduct.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+-	Anderes unethisches oder unprofessionelles Verhalten.
+
+Projektbetreuer haben das Recht und die Verantwortung, Kommentare, Commits, Code, Wiki-Bearbeitungen, Probleme und andere Beiträge zu entfernen, zu bearbeiten oder abzulehnen, die nicht mit diesem Verhaltenskodex übereinstimmen. Mit der Annahme dieses Verhaltenskodex verpflichten sich die Projektbetreuer, diese Grundsätze fair und konsequent auf jeden Aspekt der Projektleitung anzuwenden. Projektbetreuer, die den Verhaltenskodex nicht befolgen oder durchsetzen, können dauerhaft vom Projektteam ausgeschlossen werden.
+
+Dieser Verhaltenskodex gilt sowohl innerhalb von Projekträumen als auch in öffentlichen Räumen, wenn eine Person das Projekt oder seine Gemeinschaft vertritt.
+
+Fälle von missbräuchlichem, belästigendem oder anderweitig unzumutbarem Verhalten in Kubernetes können gemeldet werden, indem Sie sich an das [Kubernetes Komitee für Verhaltenskodex](https://git.k8s.io/community/committee-code-of-conduct) wenden unter <conduct@kubernetes.io>. Für andere Projekte wenden Sie sich bitte an einen CNCF-Projektbetreuer oder an unseren Mediator, Mishi Choudhary <mishi@linux.com>.
```
Comment (mkorbi) [inline content/de/community/static/cncf-code-of-conduct.md]: Fälle von missbräuchlichem, belästigendem oder anderweitig unzumutbarem Verhalten im Kubernetes Projekt können gemeldet werden,

File: content/de/community/code-of-conduct.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+<a href="https://github.com/cncf/foundation/blob/214585e24aab747fb85c2ea44fbf4a2442e30de6/code-of-conduct-languages/de.md">Commit 214585e</a> gefunden werden kann.
+Wenn Ihnen auffällt, dass die hier gezeigte Version nicht mehr aktuell ist,
```
Comment (mkorbi) [inline content/de/community/code-of-conduct.md]: Wenn dir auffällt, dass die hier gezeigte Version nicht mehr aktuell ist

File: content/de/community/code-of-conduct.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+<a href="https://github.com/kubernetes/website/issues/new">eröffnen Sie bitte ein Issue</a>.
```
Comment (mkorbi) [inline content/de/community/code-of-conduct.md]: eröffne bitte ein Issue

File: content/de/community/code-of-conduct.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+Wenn Ihnen bei einem Event, einem Meeting, in Slack oder einem anderen
```
Comment (mkorbi) [inline content/de/community/code-of-conduct.md]: Wenn dir bei

File: content/de/community/code-of-conduct.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+Kommunikationskanal ein Verstoß gegen den Verhaltenskodex auffällt, wenden Sie sich an das <a href="https://git.k8s.io/community/committee-code-of-conduct">Kubernetes Code of Conduct Committee</a>.
```
Comment (mkorbi) [inline content/de/community/code-of-conduct.md]: ..., wende dich an das <a ...>

File: content/de/community/static/cncf-code-of-conduct.md (hunk 2 above)
Comment (alexzimmer96) [reply content/de/community/static/cncf-code-of-conduct.md]: I would prefer to update this at the source:
https://github.com/cncf/foundation/blob/master/code-of-conduct-languages/de.md

**Comment by alexzimmer96** [comment]
Hi @mkorbi 
I also prefer using informal speech. I made the according changes.

Like already mentioned, i want to avoid changing the copied CoC-Version. I think we should file a PR in the CoC-Repository instead.

File: content/de/community/static/cncf-code-of-conduct.md (hunk 2 above)
Comment (mkorbi) [reply content/de/community/static/cncf-code-of-conduct.md]: sounds good, go for it :)

**Comment by mkorbi** [comment]
/lgtm 
@alexzimmer96 thank you for your contribution!

Do we have anything else to add?

**Comment by alexzimmer96** [comment]
I think this PR is complete. The updated CoC can be replaced when the PR is accepted. I don't know how long this will take.

**Pull request merged**